docs/api/rest_api/rest_api_reference/input/examples/shipping/methods/identifier/GET/ShippingMethod.xml.example

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ShippingMethod media-type="application/vnd.ibexa.api.ShippingMethod+xml" href="/api/ibexa/v2/shipping/method-types/method1">
```

Comment:
There might be something wrong here.
Like in ShippingMethodRefList, it probably should be
```suggestion
<ShippingMethod media-type="application/vnd.ibexa.api.ShippingMethod+xml" href="/api/ibexa/v2/shipping/method/method1">
```

It comes from https://github.com/ibexa/shipping/pull/25/files#diff-4ef4d687a3bdb5cdbbb1ae7f0bf7ac0a45372a32d8def7958df51b9d60d2f07aR43

@alongosz will check there if this is intended or if there was a confusion.

Reply:
See ibexa/shipping#57

Reply:
- ibexa/shipping#57 merged
- Example fixed in 9d16685ea246f72ef870a9cc6bff08f9dbd8df2d